bin/deploy-pr.sh

```diff
@@ -31,6 +33,16 @@ if [ -n "$CI_PULL_REQUESTS" ] && [ "$WEBCHANGES" -ne 0 ]; then
 
   # Push
   cf push "hitech-apd-frontend-pr$PRNUM" -n "hitech-apd-pr$PRNUM" -b https://github.com/cloudfoundry/staticfile-buildpack.git -m 64M --no-manifest -p web/dist
+
+  # Add a comment on Github, if there's not one already
+  COMMENTS=$(curl -s -u "$GH_BOT_USER:$GH_BOT_PASSWORD" https://api.github.com/repos/18f/cms-hitech-apd/issues/$PRNUM/comments | jq -c -r '.[] | {id:.id,user:.user.login}' | grep "$GH_BOT_USER")
+  if ! [ "$COMMENTS" ]; then
+    URL="https://hitech-apd-pr$PRNUM.app.cloud.gov/"
+    curl -s -u "$GH_BOT_USER:$GH_BOT_PASSWORD" -d '{"body":"See this pull request in action: '"$URL"'"}' -H "Content-Type: application/json" -X POST "https://api.github.com/repos/18f/cms-hitech-apd/issues/$PRNUM/comments"
+  fi
+
+elif [ "$WEBCHANGES" -eq 0 ]; then
+  echo "No changes to /web"
 else
-  echo "Not a pull request, or no changes to /web"
+  echo "Not a pull request"
```

Comment:
Just splitting out this `else` so the output in CircleCI tells exactly why it didn't deploy - a few times it has failed to deploy and I assume it's because the branch was tested before the PR was created, but I'd like to see something more explicit about that.